deploy/assisted-image-service.yaml

```diff
@@ -1,6 +1,11 @@
 apiVersion: v1
 kind: List
 items:
+- kind: ServiceAccount
```

Comment:
Is this committed here by mistake?

Reply:
No, this SA is required by other Deployments in `deploy/`, so while its not required to pass unit tests, its required to pass `make deploy-test` on k8s

Reply:
Fair enough, but seems a bit unrelated to this PR. It's important we keep things distinct in case they need to be backported.

Reply:
This is especially important because it's really annoying backporting things that already exist in master - you have to create a fake BZ for master without any PR, and you can't use the convenient cherry-pick robot for the process

Reply:
I don't think the whole PR is worth backporting - this is a dev-only feature, so I'd rather have all things in one PR for this.

However I don't mind breaking this up

Reply:
I meant it the other way around, that small change may be worth backporting and it will be a pain to do it if it's merged here

Reply:
good point - spinned it off to https://github.com/openshift/assisted-service/pull/3629